Mathlib/Data/Nat/Bits.lean

```diff
@@ -286,24 +286,21 @@ lemma testBit_bit_zero (b n) : testBit (bit b n) 0 = b := by
   rw [testBit, bit]
   cases b
   · simp [bit0, ← Nat.mul_two]
-  · simp only [cond_true, bit1, bit0, shiftRight_zero, and_one_is_mod, bne_iff_ne]
-    simp only [← Nat.mul_two]
-    rw [Nat.add_mod]
-    simp
+  · simp [bit0, bit1, ← Nat.mul_two]
 
 #align nat.test_bit_zero Nat.testBit_zero
 
-lemma bodd_eq_and_one_ne_zero : ∀ n, bodd n = (n &&& 1 != 0)
+lemma bodd_eq_one_and_ne_zero : ∀ n, bodd n = (1 &&& n != 0)
```

Comment:
Does this want a deprecated alias? (But also I don't really care about `bodd` :))

Reply:
Yeah, this `bodd` stuff is hard to care about. :-)